.travis.yml

```diff
@@ -43,7 +43,7 @@ before_script:
   - ./configure
 
 script:
-  - make -j2
+  - make -j$(sysctl -n hw.ncpu)
```

Comment:
It is the number of CPUs in the platform like `nprocs` in ubuntu. This pr has already been merged into master.